src/api/core/organizations.rs

```diff
@@ -352,9 +363,10 @@ async fn get_org_collections_details(org_id: &str, headers: ManagerHeadersLoose,
             })
             .collect();
 
-        if user_org.access_all {
-            assigned = true;
-        }
+        // if current user is in any collection-assigned group
+        // or in a group having access to all collections
+        // or itself has access to all collections
+        assigned = !assigned && has_collection_access_via_group.contains(&user_org.uuid);
```

Comment:
I think this is wrong. It should be 
```suggestion
        if (!assigned) {
            assigned = has_collection_access_via_group.contains(&user_org.uuid);
        }
```

Reply:
As one liner it should be `A || B` (which is only false if both A and B are false) not `!A && B` (which is only true if A is false and B is true) so you could write it as
```suggestion
        assigned = assigned || has_collection_access_via_group.contains(&user_org.uuid);
```